libs/platform/src/lib/components/split-menu-button/split-menu-button.component.html

```diff
@@ -1,13 +1,15 @@
-<div class="fd-button-split" role="group" aria-label="button-split">
+<div class="fd-button-split" role="group" aria-label="split-button" [attr.dir]="dir">
     <button
         fd-button
         [attr.id]="id"
         [attr.name]="name"
-        aria-label="buttonLabel"
+        [attr.title]="title"
         [disabled]="disabled"
         [fdType]="type"
         [glyph]="icon"
+        tabindex="0"
         [compact]="contentDensity === 'compact'"
+        [attr.aria-label]="title"
         (click)="primaryButtonClicked($event)"
```

Comment:
What about this aria-label-by do we need it too?

Reply:
while we are using title, have removed aria-label. it has same functionality as title.